src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -501,6 +501,21 @@ public static int GetInputMethodHeight(IEnumerable<AccessibilityWindowInfo> wind
             }
             return inputMethodWindowHeight;
         }
+        
+        public static bool IsAutofillServicePromptVisible(IEnumerable<AccessibilityWindowInfo> windows)
+        {
+            if (windows != null)
+            {
+                foreach (var window in windows)
+                {
+                    if (window.Title?.ToLower().Contains("autofill") ?? false)
+                    {
+                        return true;
+                    }
+                }
+            }
+            return false;
```

Comment:
can condense/simplify to:
`return windows?.Any(w => w.Title?.ToLower()?.Contains("autofill") ?? false) ?? false;`

Reply:
Still wrapping my head around Linq :)  I think I follow this one but can you confirm if the `?` after `ToLower()` is actually necessary?  I would assume if `Title` was null we would be returning false before getting to `ToLower()`.

Reply:
You're correct, the `?.` operator does short-circuit after the first null instance, in this case `?.Title`; so you could remove the second one after `ToLower()` (just a bad habit, but ToLower() would not also return `null` if we've already exited from title, just odd because `string` is not a value-type in .net but is a special snowflake type).